aws/msk_cluster_test.go

```diff
@@ -0,0 +1,122 @@
+package aws
+
+import (
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/service/kafka/types"
+	"github.com/gruntwork-io/cloud-nuke/config"
+)
+
+func TestShouldIncludeMSKCluster(t *testing.T) {
```

Comment:
Can we also test `getAllMSKClusters` and `nukeAllMSKClusters` logic as well?

Reply:
I added tests that deploy, discover, and destroy an MSK cluster, but they are slow as we need to wait for infra to be cleaned up on a delete/terminate. 

I'd be happy to rewrite it as a mocked set of tests if you'd prefer, they would be similar to the ACM tests.

Reply:
Hey @robpickerill, that would be really great. You can look at some of the refactoring work that I've been implementing and follow similar pattern for unit tests. 
* https://github.com/gruntwork-io/cloud-nuke/pull/501
* https://github.com/gruntwork-io/cloud-nuke/pull/503

Reply:
done in https://github.com/gruntwork-io/cloud-nuke/pull/489/commits/d903e0b787a975f44fd94b4c470a74af4d52a33d